tools/scoring/tests/samples/solution/real_world_solution.yaml

```diff
@@ -0,0 +1,383 @@
+US-MTV-1804:
+  id: FACILITIES/buildings/779356778
+  type: FACILITIES/BUILDING
+  guid: f1124cdb-7893-4d08-943c-ad777c081fce
+EF-1 / Electric Rm 1C1:
+  cloud_device_id: 2550389110353931
+  id: CDM/2550389110353931
+  translation:
+    run_command:
+      present_value: data.binary-output_1.present-value
+      states:
+        OFF: inactive
+        ON: active
+    run_status:
+      present_value: data.binary-input_1.present-value
+      states:
+        OFF: inactive
+        ON: active
+    zone_air_cooling_temperature_setpoint:
+      present_value: data.analog-value_7.present-value
+      units:
+        key: data.analog-value_7.present-value
+        values:
+          degrees_fahrenheit: degrees-fahrenheit
+    zone_air_heating_temperature_setpoint:
+      present_value: data.analog-value_8.present-value
+      units:
+        key: data.analog-value_8.present-value
+        values:
+          degrees_fahrenheit: degrees-fahrenheit
+    zone_air_temperature_sensor:
+      present_value: data.analog-input_1.present-value
+      units:
+        key: data.analog-input_1.present-value
+        values:
+          degrees_fahrenheit: degrees-fahrenheit
+  type: HVAC/FAN_SS_DSP
+  guid: f00744b4-489e-4dd7-803c-d7d092ec3a2c
+Shelter/Purge / 1804 Shoreline:
+  cloud_device_id: 2754543054948093
+  connections:
+    US-MTV-1804:
+    - CONTAINS
+  id: CDM/2754543054948093
+  translation:
+    outside_air_dewpoint_temperature_sensor:
+      present_value: data.analog-value_1304.present-value
+      units:
+        key: data.analog-value_1304.present-value
+        values:
+          degrees_fahrenheit: degrees-fahrenheit
+    outside_air_relative_humidity_sensor:
+      present_value: data.analog-value_1302.present-value
+      units:
+        key: data.analog-value_1302.present-value
+        values:
+          percent: percent-relative-humidity
+    outside_air_specificenthalpy_sensor:
+      present_value: data.analog-value_1303.present-value
+      units:
+        key: data.analog-value_1303.present-value
+        values:
+          btus_per_pound_dry_air: btus-per-pound-dry-air
+    outside_air_temperature_sensor:
+      present_value: data.analog-value_1301.present-value
+      units:
+        key: data.analog-value_1301.present-value
+        values:
+          degrees_fahrenheit: degrees-fahrenheit
+    outside_air_wetbulb_temperature_sensor:
+      present_value: data.analog-value_1305.present-value
+      units:
+        key: data.analog-value_1305.present-value
+        values:
+          degrees_fahrenheit: degrees-fahrenheit
+  type: HVAC/WEATHER_OA
+  guid: 0899c249-3854-40d8-ba14-2dfe64e36bcb
+power-meter-ELEV:
+  cloud_device_id: 2566753989830144
+  connections:
+    US-MTV-1804:
+    - CONTAINS
+  id: CDM/2566753989830144
```

Comment:
those must be deleted from the test file as we deprecated them, soon the parser will throw errors.

Reply:
https://github.com/google/digitalbuildings/pull/555/commits/ff694d3aaa11f680e25c31a36a3cc9a6c94230a3